charts/victoria-metrics-agent/templates/deployment.yaml

```diff
@@ -39,6 +39,7 @@ spec:
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
           image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          workingDir: /tmp
```

Comment:
this is not a backward compatible changes, so please move `workingDir` to values.yaml (the default value should be "/") and bump chart version

Reply:
you are right. I adapted it.